stardis_env3.yml

```diff
@@ -81,4 +81,4 @@ dependencies:
   
 platforms:
   - linux-64
-  - osx-64
+  - osx-64
```

Comment:
Is there any change here? Do you want to unstage this file since the before and after diffs are identical?